include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -348,10 +348,24 @@ namespace LinearAlgebra
        * After calling this method, read access to ghost elements of the
        * vector is forbidden and an exception is thrown. Only write access to
        * ghost elements is allowed in this state.
+       *
+       * @deprecated Use zero_out_ghost_values() instead.
        */
       void
       zero_out_ghosts() const;
 
+
+      /**
+       * This method zeros the entries on ghost dofs, but does not touch
+       * locally owned DoFs.
+       *
+       * After calling this method, read access to ghost elements of the
+       * vector is forbidden and an exception is thrown. Only write access to
```

Comment:
An exception is thrown *when*? I think what you say is that an exception is thrown if one tries to read from these values, but that's not clear from this one sentence (though the next clarifies it).

It's an interesting concept, though. Do any other functions impose this kind of restriction? When is the restriction lifted?